src/app/components/cards/PostsList.jsx

```diff
@@ -267,6 +268,27 @@ class PostsList extends React.Component {
         const renderSummary = items =>
             items.map((item, i) => {
                 const every = this.props.adSlots.in_feed_1.every;
+                if (this.props.videoAdsEnabled && i === 4) {
+                    return (
```

Comment:
`renderSummary` logic is not very clear; this case needs to be merged into the lower section (where ad is appended to post), otherwise the post at `i==4` will never be rendered

Reply:
moved this to an `else if` below - just in case `every` is ever a 4, this is the better way to handle it (because we wouldn't display both a video ad and a regular ad in the same slot)